src/dk/aau/cs/ds306e18/tournament/rlbot/configuration/BotConfig.java

```diff
@@ -0,0 +1,111 @@
+package dk.aau.cs.ds306e18.tournament.rlbot.configuration;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Objects;
+
+/**
+ * Class for reading from a bots config file.
+ */
+public class BotConfig {
+
+    public final static String LOCATIONS_CONFIGURATION_HEADER = "Locations";
+    public final static String NAME = "name";
+    public final static String PYTHON_FILE = "python_file";
+    public final static String LOOKS_CONFIG = "looks_config";
+
+    public final static String DETAILS_CONFIGURATIONS_HEADER = "Details";
+    public final static String DEVELOPER = "developer";
+    public final static String DESCRIPTION = "description";
+    public final static String FUN_FACT = "fun_fact";
+    public final static String GITHUB = "github";
+    public final static String LANGUAGE = "language";
+
+    private File configFile;
+    private String name;
+    private File pythonFile;
+    private File looksConfig;
+    private String developer;
+    private String description;
+    private String funFact;
+    private String github;
+    private String language;
+
+    public BotConfig(File configFile) throws IOException {
+        this.configFile = configFile;
+        ConfigFile config = new ConfigFile(configFile);
+
```

Comment:
```suggestion

// Ensuring BotConfig has required fields available
```

Just a bit of documentation